tests/playwright/sanity/assets/dev/js/editor/elements/views/widget.test.js

```diff
@@ -0,0 +1,16 @@
+const { test, expect } = require( '@playwright/test' );
+const WpAdminPage = require( '../../../../../../../pages/wp-admin-page' );
+
+test( 'Empty widget placeholder sanity test', async ( { page }, testInfo ) => {
```

Comment:
@hein-obox Please change the title. Maybe:
"Check if empty placeholder is displayed in Image Carousel"

Reply:
Changed